misc/python/materialize/mzcompose/__init__.py

```diff
@@ -623,15 +624,36 @@ def up(
             self.compose = old_compose
             self._write_compose()
 
-    def down(self, destroy_volumes: bool = True, remove_orphans: bool = True) -> None:
+    def sanity_restart_mz(self) -> None:
```

Comment:
How long will this step roughly take? Is it in the range of 10-20 seconds or longer? If I understood correctly, it will apply to all builds?

Reply:
@philip-stoev Do you think this is worth the overhead? It hasn't actually unveiled any new bugs it seems.

Reply:
I think it is totally worth it for the following reasons:
- a lot of recent incidents have been about the inability to properly restart after the catalog or source reach a certain state. So we need more restarts in our CI
- a lot of workflows should have performed such a restart when initially written but humans frequently forget to do that. If a feature is given a Platform Check, that would have performed a restart, but humans forget to add platform checks too. 
- if an Mz restart is slow in a given situation, such as the "RestartEntireMz" scenario, it is a bug in the product -- we aim for very fast restarts otherwise our uptime is negatively impacted.